navbar.php

```diff
@@ -12,7 +12,9 @@
 		<div class="collapse navbar-collapse" id="navbarNavDropdown">
 			<ul class="navbar-nav">
 				<li class="nav-item">
-					<a class="nav-link" href="about.php">ABOUT US</a>
+					<a class="nav-link" href="about.php">
+						ABOUT US
+					</a>
```

Comment:
why are we adding the extra lines?

Reply:
I thought that was the proper format for anchor tags, will remove